### ISSUE TITLE
Troubleshooting installation

### ISSUE THREAD

**Issue by harsh-shukla** [jkimlab/mySyntenyPortal#6]
Hi Jongin,

   Thank you so much for the wonderful tool. I was successfully able to install and run mySyntentPortal in my PC (Ubuntu 16.04) and I must say its quite helpful tool for visualization.

I installed it as per instructions and was able to built my own website. But somehow the SynCircos and SynBrowser web pages were not loading through my apache2 server. Finally I figured out the problem.
There is function called split() being called in the php scripts and that was throwing an error. On further digging I found that split() function was removed in  php 7.x release (in favor of explode() ) and I had that particular version connected to my apache2. I finally installed php 5.x (split() is also deprecated in 5.x release) and linked it to my apache2 server. It finally solved the error and I could look at the plots. If someone faces this kind of error this is the solution.

Also I found the syntenic blocks using Heng Li latest aligner minimap2 and it is extremely extremely fast (it aligned two mammalian genomes on a server node with 32 processors  and 256 Gb of memory within few minutes). Any chance you could incorporate it into the pipeline.Would be quite helpful.

Once again kudos on making the tool.

Regards,
Harsh

**Comment by jongin333** [comment]
Dear Harsh,

I am very appreciated to your kindness.

I updated all the 'split()' to 'preg_split()'. The 'preg_split()' function works well in both php5 and 7 release.

In the case of the minimap2, It may be very helpful to advance the performance and usability of mySyntenyPortal. I will consider your suggestion.

Thank you very much, again.

Best regards,
Jongin

**Comment by harsh-shukla** [comment]
Dear Jongin,

   Glad I could help a little. Will be looking forward to any new updates.

Regards,
Harsh

**Comment by mictadlo** [comment]
Hi @harsh-shukla,
I created a Docker and Singularity file [here](https://github.com/jkimlab/mySyntenyPortal/issues/5#issuecomment-451027002) to get mySyntenyPortal running on HPC. 

By any chance, does your web part is inside in any container?

Thank you in advance,

Best wishes,

Michal

**Comment by harsh-shukla** [comment]
Hi Michał,

Sorry for the late reply. No , I have not used any container to install mySyntenyPortal on my HPC.

I installed mySyntenyPortal on both my server(SGE cluster) and my local machine (running Ubuntu 16.04). 

I used the server for doing the whole genome alignment as Jongin suggested #4 and than transfered the synteny files thus generated to my PC . Once transfered I built a website using the synteny files to visualize the alignment. (since apcahe2 was there only in my local machine)

P.S : There were some tweaks I had to do to the code for running the whole genome alignment on the server since I had a local copy of perl installed and there were some issues while executing. But finally I did manage to get it running.

With regards,
Harsh
